getAuthUrl does not return a promise, and causes Typescript compilation errors

See https://github.com/esteban-uo/picasa/blob/master/src/picasa.js#L129

`getAuthUrl` directly returns a String, but the doc mentions a Promise. 

Using the code as follows : 

```
app.get('/login', (req, res) => {
    const authURL = picasa.getAuthURL(googleConfig);
    res.redirect(authURL);
});
```

raises the following Typescript compilation error:

```
scripts/loginAndGetGoogleToken.ts:21:18 - error TS2345: Argument of type 'Promise<any>' is not assignable to parameter of type 'string'.

21     res.redirect(authURL);
                    ~~~~~~~
```

Because https://github.com/Microsoft/TypeScript/issues/19139 is still open, there is no way to fix this without changing the documentation. 


The doc for this method should be changed, and mention that it returns a String instead.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "picasa",
-  "version": "1.0.7",
+  "version": "1.0.8",
   "engines": {
     "node": ">=4.8.7"
   },
@@ -26,6 +26,9 @@
     "node"
   ],
   "author": "Esteban Uscanga Olea <uscanga47@gmail.com> (https://github.com/esteban-uo)",
+  "contributors": [
+    "Julien Lengrand-Lambert <julien@lengrand.fr> (http://lengrand.fr)"
+  ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/esteban-uo/picasa/issues"

--- a/src/picasa.js
+++ b/src/picasa.js
@@ -126,7 +126,7 @@ class Picasa {
    * @param {string} config.clientId
    * @param {string} config.redirectURI - URL to user will be redirected. After google displays a consent screen to the user, user will be redirect to this URL with a `code` in the URL
    * @param {function} callback - (error, response). If not provided, a promise will be returned
-   * @returns {Promise}
+   * @returns {string}
    */
   getAuthURL(config) {
     const authenticationParams = {

